Pinta.Effects/Effects/RadialBlurEffect.cs

```diff
@@ -102,11 +102,7 @@ private static ColorBgra GetFinalPixelColor (
 			X: (pixel.coordinates.X << 16) - settings.fcx,
 			Y: (pixel.coordinates.Y << 16) - settings.fcy);
 
-		int sr = sourcePixel.R;
-		int sg = sourcePixel.G;
-		int sb = sourcePixel.B;
-		int sa = sourcePixel.A;
-		int sc = 1;
+		ColorBgra.Blender blender = new (sourcePixel);
```

Comment:
I'm not 100% sure if I like the new constructor - just doing `blender += sourcePixel` feels more straightforward so that samples are always added the same way